doc/rst/language/spec/interoperability.rst

```diff
@@ -203,35 +203,23 @@ and external code.
 Referring to Standard C Types
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-In Chapel code, all standard C types must be expressed in terms of their
-Chapel equivalents. This is true, whether the entity is exported,
-imported or private. Standard C types and their corresponding Chapel
-types are shown in the following table.
-
-=========== =========== ============ ============ ======== ===========
-C Type      Chapel Type C Type       Chapel Type  C Type   Chapel Type
-=========== =========== ============ ============ ======== ===========
-int8_t      int(8)      uint8_t      uint(8)      \_real32 real(32)
-int16_t     int(16)     uint16_t     uint(16)     \_real64 real(64)
-int32_t     int(32)     uint32_t     uint(32)     \_imag32 imag(32)
-int64_t     int(64)     uint64_t     uint(64)     \_imag64 imag(64)
-chpl_bool   bool        const char\* c_string             
-\_complex64 complex(64) \_complex128 complex(128)         
-=========== =========== ============ ============ ======== ===========
-
-Standard C types are built-in. Their Chapel equivalents do not have to
-be declared using the ``extern`` keyword.
-
-In C, the “colloquial” integer type names ``char``, ``signed char``,
-``unsigned char``, (``signed``) ``short`` (``int``), ``unsigned short``
-(``int``), (``signed``) ``int``, ``unsigned int``, (``signed``) ``long``
-(``int``), ``unsigned long`` (``int``), (``signed``) ``long long``
-(``int``) and ``unsigned long long`` (``int``) may have an
-implementation-defined width. [5]_. When referring to C types in a
-Chapel program, the burden of making sure the type sizes agree is on the
-user. A Chapel implementation must ensure that all of the C equivalents
-in the above table are defined and have the correct representation with
-respect to the corresponding Chapel type.
+When referring to standard C types within Chapel code, they must
+either be expressed:
+
+* using the type aliases provided by the :mod:`CTypes` module,
+
+* using an external type declaration (see the next section), or
+
+* in terms of their Chapel equivalents (e.g., C ``double`` corresponds
+  to Chapel ``real(64)``)
+
+  .. note::
+
+     Note that a challenge to this last approach is that certain C
+     types—particularly integers—can vary in size between
+     implementations, making the second approach safer from a
```

Comment:
can it say instead "implementations. As a result, the approach of using type aliases from :mod:`CTypes` is safer from a portability perspective".

Reply:
Did a variation on this in https://github.com/chapel-lang/chapel/pull/19453/commits/c552ed22ded6775966a874162e49f45297aa4b7a (main difference being to add a qualifier since we don't support all possible extern C types in CTypes).